### PR TITLE
Security fixes, dependency upgrades, NPM -> Yarn

### PR DESCRIPTION
**Pull request by adewinter**
* Upgraded several package.json dependencies to their latest versions
* Switched from NPM to Yarn based lockfile
* Yarn/NPM audit now raises zero issues
* Modified .babelrc to support latest babel version

### CODE REVIEW

**Comment by adewinter** [comment]
Okee Dokey Karaoke, looks like the build passed so hopefully this can get merged? Thanks for taking a look!

File: foo [hunk 1]
```diff
@@ -0,0 +1,3 @@
+last 5 versions
+safari >= 7
+not dead
```
Comment (Detect) [inline foo]: is this file needed? also, different than `.babelrc` (`not dead`)

Comment (adewinter) [reply foo]: @)#(@)(#!! Thanks for catching that. Nope! Not needed.

**Comment by adewinter** [comment]
@Detect I thought about it but reckon it'll be better to do that in a different PR.  The scope for this PR is already kinda big

File: package.json [hunk 2]
```diff
@@ -56,43 +56,60 @@
...
     "http-server": "^0.11.1",
-    "jest": "^22.4.3",
+    "jest": "^24.7.1",
     "jest-fetch-mock": "^1.6.5",
     "minimist": "^1.2.0",
     "nock": "^9.2.6",
     "regenerator-runtime": "^0.11.1",
     "replace-in-file": "^3.4.0",
-    "testcafe": "^0.21.1",
+    "testcafe": "^1.1.3",
```
Comment (Detect) [inline package.json]: curious why does p2kt need testcafe?

Comment (adewinter) [reply package.json]: I believe some of their automated tests make use of them (check them out [here](https://github.com/ProdPerfect/prodperfect-keen-tracking.js/tree/master/test/testcafe))
